plone/app/widgets/dx.py

```diff
@@ -70,13 +66,6 @@
 import json
 
 try:
-    from plone.app.contenttypes.behaviors.collection import ICollection as IDXCollection  # noqa
-    from plone.app.contenttypes.behaviors import richtext  # noqa
```

Comment:
Well, I've tested this in a contentypes/widgets Project here an it worked out well. Due to the fact that PAC also uses IRichText from plone.app.textfield in the richtext behavior we can simply Register the RichWidget for this Interface and everything works automatically. But please correct me if I'm wrong. Thinking out loud further we could do the same with the query widget and get rid of contenttypes dependency?